packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -369,6 +386,17 @@ const NewNetwork = decorate([
                             <Icon icon='info' /> {_('encryptionWarning')}
                           </em>
                         </div>
+                        <label>{_('newNetworkPreferredCenter')}</label>
+                        <SelectHost
+                          onChange={effects.onChangeCenter}
+                          predicate={hostPredicate}
```

Comment:
What's behavior will lead selecting a host center then modifying the selected networks' pools to a different pool?

Reply:
If you remove the pool of the preferredCenter, the UI show it's invalid :
![invalid-center](https://user-images.githubusercontent.com/12529066/82341555-e1570a00-99f0-11ea-8a9f-8e2f10281e7b.png)

Reply:
The network can be created in this case?

Reply:
No an error will appear